packages/tezos-ts-rpc/src/tezos-ts-rpc.ts

```diff
@@ -166,4 +168,39 @@ export class RpcClient {
       key
     );
   }
+
+  /**
+   *
```

Comment:
Add description? At a minimum we can copy the description from the RPC docs and in some cases improve.

"Everything about a delegate."

"Fetches information about a delegate from RPC"